tests/test_cli.py

```diff
@@ -1542,3 +1542,25 @@ def test_tell_with_nan() -> None:
         assert len(study.trials) == 1
         assert study.trials[0].state == TrialState.FAIL
         assert study.trials[0].values is None
+
+
+@pytest.mark.skip_coverage
+@pytest.mark.parametrize(
+    "verbosity, expected",
+    [
+        ("--verbose", True),
+        ("--quiet", False),
+    ],
+)
+def test_configure_logging_verbosity(verbosity: str, expected: bool) -> None:
+
+    with StorageSupplier("sqlite") as storage:
+        assert isinstance(storage, RDBStorage)
+        storage_url = str(storage.engine.url)
+
+        # Create study.
+        args = ["optuna", "create-study", "--storage", storage_url, verbosity]
+        # If `--verbose`, the INFO level log message should be generated.
```

Comment:
Your unit tests seem OK, but could you clarify the logging level on the comment? The logging level with the `--verbose` option is `DEBUG`. I'm not sure it is easy to test `DEBUG` level logging messages because Optuna has a small number of debug messages.

Reply:
This unit test confirms the necessary condition that the behavior changes depending on the value of the `--verbose` option. As you point out, it doesn't fully test for the `DEBUG` level - sorry I couldn't think of simple way to generate `DEBUG` logging message via CLI.

Reply:
> sorry I couldn't think of simple way to generate DEBUG logging message via CLI.

I agree it is not easy to test.

> If `--verbose`, the INFO level log message should be generated.

This comment seems confusing about the log level with the `--verbose` option. How about

```python
# `--verbose` makes the log level DEBUG.
# `--quiet` makes the log level WARNING.
```

Reply:
Thank you so much for the suggestion, I have updated the comment. Kindly let me know if anything else.